var/spack/repos/builtin/packages/pika-algorithms/package.py

```diff
@@ -51,6 +51,7 @@ class PikaAlgorithms(CMakePackage):
     depends_on("pika@0.11:0.12", when="@0.1.1")
     depends_on("pika@0.11:0.15", when="@0.1.2")
     depends_on("pika@0.11:0.16", when="@0.1.3")
+    depends_on("pika@0.11:0.20", when="@0.1.4")
```

Comment:
nitpick: could drop the repeated lowerbound in the list of forward compat bounds, since it's already applied. `depends_on("@:0.20", when="@0.1.4")`

Reply:
Yep. Changed it to one general constraint with 0.11-0.20, and then separately only kept upper bounds for specific versions. Do you think this looks better?